### PR TITLE
mononoki: init at 1.2

### PR DESCRIPTION
**Pull request by christianlask**
###### Things done
- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/data/fonts/mononoki/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    url = "https://github.com/madmalik/mononoki/releases/download/${version}/mononoki.zip";
+    sha256 = "0n66bnn2i776fbky14qjijwsbrja9yzc1xfsmvz99znvcdvflafg";
+  };
+
+  buildInputs = [ unzip ];
+  phases = [ "unpackPhase" ];
+
+  unpackPhase = ''
+    mkdir -p $out/share/fonts/truetype/mononoki
+    unzip $src -d $out/share/fonts/mononoki
```
Comment (vcunat) [inline pkgs/data/fonts/mononoki/default.nix]: You're unzipping somewhere else than into the new directory.

**Comment by hiberno** [comment]
@vcunat Thanks. Fixed.

**Comment by vcunat** [comment]
The font seems nice :-)

**Comment by hiberno** [comment]
@vcunat I think so, too. That's why I tried to package it.

**Pull request merged**